docs/page.css

```diff
@@ -40,11 +40,11 @@ h3 {
 	font-size: 16px;
 	font-weight: normal;
 
-	margin-top: 30px;
+	margin-top: 40px;
```

Comment:
I agree that the docs could do with some minor styling updated in places, but it would be better to include these as a seperate PR.